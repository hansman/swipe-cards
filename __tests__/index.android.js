import 'react-native';
import React from 'react';
import Index from '../index.android.js';
import renderer from 'react-test-renderer';

describe('Android', () => {

    it('renders correctly', () => {
        const tree = renderer.create(
            <Index />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });

});
