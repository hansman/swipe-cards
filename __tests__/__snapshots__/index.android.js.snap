exports[`Android renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(0, 0, 0, 0.1)",
      "flex": 1,
    }
  }>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail">
      Loading cards...
    </Text>
  </View>
</View>
`;
